server/controllers/gh.js

```diff
@@ -28,8 +28,11 @@ function getIssues(req, res) {
 }
 
 function getIssue(req, res) {
-    var issueRequestUrl = issuesRequestUrl + '/' + req.params.id;
+    _getIssueBody(req, res);
```

Comment:
you may just check if it's an XHR request (`req.xhr`) and use `{ block: 'issues' }` as a context. so no need in `context` param at all